src/main/java/uc/user/logout/LogoutInputBoundary.java

```diff
@@ -1,9 +1,14 @@
 package uc.user.logout;
 
 /**
- * Defines a logout method to be called by the Controller and implemented by the Use Case Interactor.
+ * LoginOutputBoundary provides an entry-point to the logout use case.
+ * Used by a Controller and implemented by LogoutInteractor.
```

Comment:
I would be cautious about mentioning anything in the layer above within documentation to ensure that it remains CLEAN but I think in this instance it is ok since we always expect a controller in layer above